source/statements.tex

```diff
@@ -898,7 +898,9 @@
 \indextext{initialization!jump past}%
 \indextext{\idxcode{goto}!initialization and}%
 It is possible to transfer into a block, but not in a way that bypasses
-declarations with initialization. A program that jumps\footnote{The transfer from the condition of a \tcode{switch} statement to a
+declarations with initialization (including such in \grammarterm{condition}s
```

Comment:
I find this use of "such" to be a little awkward.

Reply:
Let's try "ones" instead; Google found a similar sentence "Dozens of Vegetable Products, Including Ones Sold at Trader Joe's and Walmart, Recalled Over Listeria Concerns".

Reply:
Yes, I like that more. Another option that doesn't sound wrong to me is "including such declarations in ..."